src/contacts/contacts.py

```diff
@@ -24,18 +24,44 @@
 import sys
 
 import jwt
+from easy_profile import EasyProfileMiddleware
+from easy_profile.reporters import Reporter
 from flask import Flask, jsonify, request
 import bleach
-from opentelemetry import trace
+from opentelemetry import metrics, trace
```

Comment:
In the contacts service, I tried to instrument metrics reported by the sqlalchemy-easy-profile using opentelemetry. However, the Cloud Monitoring exporter doesn't fully support all metric types yet